internal/download/artifact.go

```diff
@@ -19,3 +24,11 @@ func ShouldDownloadArtifact(jobID string, passed, timedOut bool, async bool, cfg
 
 	return false
 }
+
+// Cleanup removes previous downloaded artifacts
+func Cleanup(directory string) {
+	err := os.RemoveAll(directory)
+	if err != nil {
+		log.Err(err).Msg("Unable to cleanup previous artifacts ()")
```

Comment:
Not sure what the parentheses are for? 🤔 

```suggestion
		log.Err(err).Msg("Unable to cleanup previous artifacts")
```

Reply:
My bad. removed